test/StakeManager.t.sol

```diff
@@ -392,31 +573,97 @@ contract ExecuteAccountTest is StakeManagerTest {
             }
         }
     }
-}
 
-contract UserFlowsTest is StakeManagerTest {
-    function setUp() public override {
-        StakeManagerTest.setUp();
-    }
+    function test_ShouldNotMintMoreThanCap() public {
+        uint256 stakeAmount = 10_000_000;
+        deal(stakeToken, testUser, stakeAmount);
 
-    function test_StakedSupplyShouldIncreaseAndDecreaseAgain() public {
-        // ensure users have funds
-        deal(stakeToken, testUser, 1000);
-        deal(stakeToken, testUser2, 1000);
+        userVaults.push(_createStakingAccount(makeAddr("testUser"), stakeAmount, 0));
+        userVaults.push(_createStakingAccount(makeAddr("testUser2"), stakeAmount, stakeManager.MAX_LOCKUP_PERIOD()));
+        userVaults.push(_createStakingAccount(makeAddr("testUser3"), stakeAmount, stakeManager.MIN_LOCKUP_PERIOD()));
 
-        StakeVault userVault = _createTestVault(testUser);
-        StakeVault user2Vault = _createTestVault(testUser2);
+        for (uint256 i = 0; i < 209; i++) {
```

Comment:
It would be great if we could put a name on the `209` as it looks like some arbitrary thing, but it's not.